Module.yaml

```diff
@@ -1,38 +1,54 @@
+    options: []                         # Options for the field if it is a select or multiple select. An array of string options
+    dependencies: []                    # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
+      - key:                            # key of the dependency
+        value:                          # value of the dependency
+  - name: Data type                     # name of the field to be displayed on the UI.
+    key: DATA_TYPE                      # key to set in the environment variables.
+    description: byte or word           # Description/helper text for the field.
+    value: ''                           # Value to be set in the environment variables.
+    type: 'select'                      # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
+    options:
+      - byte
+      - word
+    dependencies: []
+  - name: I2C slave address             # name of the field to be displayed on the UI.
+    key: SLAVE_ADDR                     # key to set in the environment variables.
+    description: I2c slave address      # Description/helper text for the field.
+    value: ''                           # Value to be set in the environment variables.
+    type: 'number'                      # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
+    options: []                         # Options for the field if it is a select or multiple select. An array of string options
+    dependencies: []                    # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
+      - key:                            # key of the dependency
+        value:                          # value of the dependency
+  - name: Period             # name of the field to be displayed on the UI.
+    key: PERIOD                     # key to set in the environment variables.
+    description: The I2C interface read data every choosen period (in seconds)      # Description/helper text for the field.
+    value: ''                           # Value to be set in the environment variables.
+    type: 'number'                      # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
+    options: []                         # Options for the field if it is a select or multiple select. An array of string options
+    dependencies: []                    # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
+      - key:                            # key of the dependency
+        value:                          # value of the dependency
 ports:                                  # Array of ports that will be exposed by the module.
```

Comment:
no need to mount any ports or directories

Reply:
yes we needn't to mount any port because i2c integrated , but we have give container access to /dev/i2c1 interface

Reply:
@GhasBarbouchi as far as I see, you are still binding the ports. why did you decided to keep that?